dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/view/HostView.java

```diff
@@ -0,0 +1,21 @@
+package com.dotcms.rest.api.v1.secret.view;
+
+public class HostView {
+
+    private String hostId;
+    private String hostName;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'hostName' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4623512)